autoload/flake8.vim

```diff
@@ -184,8 +198,13 @@ function! s:Flake8()  " {{{
     endif
 endfunction  " }}}
 
-"" markers
 
+function! s:MakeMap(results)  " {{{
+    for result in a:results
+    endfor
+endfunction  " }}}
```

Comment:
Is this still called anywhere, or could this be removed?

Reply:
No. Looks like I forgot to remove it. Made the necessary changes now

Reply:
Perfect, thanks!